templates/tenant.yaml.in

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: keylime-tenant
+  namespace: REPLACE_KEYLIME_NAMESPACE
+  labels:
+    app: keylime
+spec:
+  selector:
+    matchLabels:
+      app: keylime
+      role: tenant
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: keylime
+        role: tenant
+    spec:
+      containers:
+      - name: registrar
+        image: REPLACE_KEYLIME_IMAGE_PREFIX/keylime_registrar
```

Comment:
This should probably use the keylime_tenant image

Reply:
ooops, my mistake (the image is basically the same, but we should not count on it)

Reply:
argh, multitasking, with the usual levels of success and precision :joy: